src/Customers/Customer.php

```diff
@@ -426,6 +426,17 @@ public function getEmails(): Collection
         return $this->emails;
     }
 
+    /**
+     * @return null|string
+     */
+    public function getFirstEmail(): ?string
+    {
+        $emails = $this->emails->toArray();
+        $email = array_shift($emails);
+
+        return $email ?? null;
```

Comment:
The `toArray()` method returns an array of `Email` objects. If that behaviour is expected, this should be `return $email->value ?? null;`

Reply:
@miken32 I've updated this. The `ConversationTest` was passing a string as an email when creating the collection to test the `extract` method. I'm fairly sure this is where the simply `$email ?? null` check came from. 

After updating that test to pass an `Email` entry object into the collection, it became apparent what the solution should be in the `Customer::getFirstEmail` method. 

Thanks for pointing this out.